### DISCUSSION TITLE
How to Free Deploy on Render.com 🆓

### DISCUSSION THREAD

**Discussion by Malith-Rukshan** [Malith-Rukshan/Auto-Reaction-Bot#21] [Q&A] [answered]
How to Deploy for Free on Render.com ?

**Comment by Malith-Rukshan** [comment, marked as answer]
# How to Deploy for Free on Render.com
Follow these steps to deploy this project for free on Render.com:

1. **Sign Up or Log In**: Go to [Render.com](https://render.com/) and sign up for a free account or log in if you already have one.
2. **Create a New Web Service**: Click on New > Web Service.
3. **Connect Your Repository**: Link your GitHub repository to Render.com. _(Also you can copy past this public repo link)_
4. **Follow the Video Guide**: Refer to the video below for detailed instructions on how to fill in the required details.

https://github.com/user-attachments/assets/68e98443-df35-4e74-a104-e3fe41348187

Feel free to reach out in the discussion section if you have any questions!

**Comment by aman890890** [comment]
good

**Reply by aman890890** [reply]
https://api.telegram.org/bot<YourBotToken>/getWebhookInfo

what the result if my bot in good?
